src/reader-activation/index.js

```diff
@@ -422,6 +402,60 @@ function attachNewsletterFormListener() {
 	} );
 }
 
+
+/**
+ * Checkout functions.
+ */
```

Comment:
Given that this set of functions is expanding for checkout-specific handling, would it make sense to move it to a separate `checkout.js`?

Reply:
Great idea! Done in https://github.com/Automattic/newspack-plugin/pull/3343/commits/f68ba174858d8213dacda9f734355bd60f9fd638